components/src/core/Drawer/DrawerNavItem/DrawerNavItem.tsx

```diff
@@ -194,34 +198,35 @@ const DrawerNavItemRender: React.ForwardRefRenderFunction<HTMLElement, DrawerNav
     const { open: drawerOpen = true, activeItem, onItemSelect } = useDrawerContext();
     const { activeHierarchy } = useNavGroupContext();
     const previousActive = usePrevious(activeItem);
+    const colorScheme = useColorScheme();
 
     // Primary color manipulation
     const fivePercentOpacityPrimary = color(
-        theme.palette.mode === 'dark' ? theme.palette.primary.dark : theme.palette.primary.main
+        colorScheme.mode === 'dark' ? theme.palette.primary.dark : theme.palette.primary.main
```

Comment:
Same idea...can we find a way to use applyStyles to avoid relying on useColorScheme?